.dialyzer_ignore

```diff
@@ -0,0 +1,3 @@
+lib/lob/intl_verification.ex:6: Function resource_url/1 will never be called
```

Comment:
I think we can do this for now. Given that `resource_url/1` is added through a macro, what do you think of adding an option to conditionally remove the `resource_url/1` definition (opt-in by default) such that we don't have these ignores to begin with.

```elixir
defmodule Lob.ResourceBase do

  defmacro __using__(opts) do
    options = Keyword.fetch!(opts, :options)

    quote do

      unless :ignore_resource_url in unquote(options), do
        @spec resource_url(String.t) :: String.t
        defp resource_url(resource_id), do: "#{base_url()}/#{resource_id}"
      end

    end

  end

end

defmodule Lob.IntlVerification do

  use Lob.ResourceBase, endpoint: "intl_verifications", methods: [], options: [:ignore_resource_url]

end

```

Reply:
Yeah that seems like a good compromise. I think its still a bit esoteric but its better than ignoring a legitimate warning.

Reply:
Ticketed up as ENG-5695